app/src/main/java/swati4star/createpdf/util/FileUtils.java

```diff
@@ -486,4 +489,46 @@ else if (scheme.equals("content")) {
         }
         return fileName;
     }
+
+    /**
```

Comment:
This file already has a lot of utils functions. Move this function to a new file `PageSizeUtils.java`